autoscaling/fuel-core-cpu-hpa.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: autoscaling/v1
+kind: HorizontalPodAutoscaler
+metadata:
+  annotations:
+  name: fuel-core-cpu-hpa
+  namespace: fuel-core
+spec:
+  maxReplicas: 3
+  minReplicas: 1
```

Comment:
We may want this minimum to be at least 2 in production environments in order to avoid a long outage in case we are scaled down to one node and it crashes.